### ISSUE TITLE
Renderer or template is not applied after inline editing in grid

### ISSUE THREAD

**Issue by mvdputten** [atatanasov/gijgo#39]
When a renderer or a template is configured for a column in the grid, I would expect it to be applied not just when first rendering the cell, but also after editing the cell.
Currently this is not the case.
I ran into this issue when I implemented a renderer and (picklist)editor to display human readable text but work internally with numerical values.

**Comment by mvdputten** [comment]
I've written an example to show the current behaviour:
```html
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
	<head>
		<title>atvise - next generation</title>
		<meta charset="utf-8" />
		<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
		<script src="http://code.gijgo.com/1.0.0/js/gijgo.js" type="text/javascript"></script>
<!--
		<script src="http://rawgit.com/mvdputten/gijgo/bugfixrenderafteredit/src/widget.js" type="text/javascript"></script>
		<script src="http://rawgit.com/mvdputten/gijgo/bugfixrenderafteredit/src/grid/js/grid.base.config.js" type="text/javascript"></script>
		<script src="http://rawgit.com/mvdputten/gijgo/bugfixrenderafteredit/src/grid/js/grid.base.events.js" type="text/javascript"></script>
		<script src="http://rawgit.com/mvdputten/gijgo/bugfixrenderafteredit/src/grid/js/grid.base.methods.js" type="text/javascript"></script>
		<script src="http://rawgit.com/mvdputten/gijgo/bugfixrenderafteredit/src/grid/js/grid.base.widget.js" type="text/javascript"></script>
		<script src="http://rawgit.com/mvdputten/gijgo/bugfixrenderafteredit/src/grid/js/grid.inlineEditing.js" type="text/javascript"></script>
-->
    <link href="http://code.gijgo.com/1.0.0/css/gijgo.css" rel="stylesheet" type="text/css" />
	</head>
	<body>
		<table id="grid" style="width:100%"/>
		<script type="text/javascript">
    
    var weekdays = ["Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"]
        $(document).ready(function () {
            var data, textEditor, weekDayEditor, weekDayRenderer;
            data = [
                { "ID": 1, "Name": "Hristo Stoichkov", "DayOfWeek": 1 },
                { "ID": 2, "Name": "Ronaldo Luís Nazário de Lima", "DayOfWeek": 3 },
                { "ID": 3, "Name": "David Platt", "DayOfWeek": 4 }
            ];
            textEditor = function($container, currentValue) {
                
                $container.append('<input type="text" value="' + currentValue.replace(/\'(.*)\'/, '$1') + '" style="width:95%"/>');
            },
            weekDayRenderer = function (value, record, $cell, $displayEl) {
              $displayEl.text(weekdays[value]);
            };
            weekDayEditor = function ($container, currentValue) {
                var selectHtml, selected;
                selectHtml = "<select>";
                weekdays.forEach(function (label, value) {
                  selected = (currentValue !== undefined && value === currentValue)?"SELECTED":"";
                  selectHtml += "<option " + selected + " value=" + value + ">" + label + "</option>"
            });
                selectHtml += "</select>";
                $container.append(selectHtml);
            };
            $("#grid").grid({
                dataSource: data,
                columns: [
                    { field: "ID", width: 34 },
                    { field: "Name", tmpl: "'{Name}'", title: "Template: '{Name}'", editor: textEditor },
                    { field: "DayOfWeek", title: "Renderer: Day of week", renderer:weekDayRenderer, editor: weekDayEditor },
                ]
            });
        });

    </script>
	</body>
</html>
```

**Comment by atatanasov** [comment]
Thanks for finding this issue. We are going to fix this problem in the new release, that is planned for release later this year.

**Comment by atatanasov** [comment]
This issue is fixed in the new development version 1.1.0. You can see an working example at http://localhost:59034/Temp/2dc31c65-e4e7-44b6-8bae-36b5feff16de.html

You can use the current development version from:
https://cdn.rawgit.com/atatanasov/gijgo/master/dist/combined/js/gijgo.min.js
https://cdn.rawgit.com/atatanasov/gijgo/master/dist/combined/css/gijgo.min.css

**Comment by mvdputten** [comment]
Works like a charm. Thanks!
